webpush_channels/views/channels.py

```diff
@@ -46,8 +47,14 @@ def send_push_notifications(request):
         subscriptions += user_subscriptions
 
     for subscription in subscriptions:
-        push_initialize = WebPusher(subscription)
-        push_initialize.send(data=request.validated.get('data'), ttl=15)
+        data = request.validated.get('data')
```

Comment:
WebPush is supposed to handle empty payload so we should support that case rather than adding a default payload.